Packs/Legacy/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py

```diff
@@ -170,6 +173,44 @@ def test_similar_context(mocker):
 
     args.update({'similarIncidentFields': 'name', 'similarContextKeys': 'listValue.name'})
     result = main()
+    assert len(result['EntryContext']['similarIncidentList']) == 1
+    assert result['EntryContext']['similarIncidentList'][0]['rawId'] == 2
+
+    args.update({'similarIncidentFields': 'name', 'similarContextKeys': 'dictListValue.test'})
+    result = main()
     assert len(result['EntryContext']['similarIncidentList']) == 2
     assert result['EntryContext']['similarIncidentList'][0]['rawId'] == 3
     assert result['EntryContext']['similarIncidentList'][1]['rawId'] == 2
+
+    args.update({'similarIncidentFields': 'name', 'similarContextKeys': 'dictListValue'})
+    result = main()
+    assert len(result['EntryContext']['similarIncidentList']) == 1
+    assert result['EntryContext']['similarIncidentList'][0]['rawId'] == 2
+
```

Comment:
Please add one more test for empty condition to context - means that the incident does not have the context key that specified in the SimilarContextKeys. This test should return error